ArduCopter/RC_Channel_Copter.cpp

```diff
@@ -764,24 +779,25 @@ void RC_Channels_Copter::auto_trim_run()
         }
 
         // only trim in certain modes:
-        if (copter.flightmode != &copter.mode_stabilize &&
-            copter.flightmode != &copter.mode_althold) {
+        if (!copter.flightmode->allows_auto_trim()) {
             auto_trim_cancel();
             return;
         }
 
         // must be started and stopped mid-air:
         if (copter.ap.land_complete_maybe) {
+            GCS_SEND_TEXT(MAV_SEVERITY_WARNING,"Must be flying to use AUTOTRIM");
             auto_trim_cancel();
             return;
         }
-
         // calculate roll trim adjustment
         float roll_trim_adjustment = ToRad((float)get_roll_channel().get_control_in() / 4000.0f);
+        roll_trim_adjustment = ToRad(float(copter.attitude_control->get_att_target_euler_cd().x / 2000.0f));
```

Comment:
This makes no sense - you're just blatting the previously assigned value.

Reply:
yep...missed removing that...was there in case it didn't work I would not have to remember what current was